package.json

```diff
@@ -35,9 +35,9 @@
     "url": "https://github.com/edx/frontend-app-communications/issues"
   },
   "dependencies": {
-    "@edx/brand": "npm:@edx/brand-edx.org@^2.0.3",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
```

Comment:
I thought we would want the Open edX branded version of the package by default. We will override this dependency in an upcoming `edx-internal` PR following these changes.